src/gluonts/dataset/split/splitter.py

```diff
@@ -107,7 +109,7 @@ def from_data_entry(
     def to_data_entry(self) -> DataEntry:
         ret = {
             FieldName.START: self.start,
-            FieldName.ITEM_ID: self.item,
+            FieldName.ITEM_ID: self.item_id,
```

Comment:
Thanks for the changes! just a minor comment: no need to add `item_id` if its None. You can just remove that line and add after the assignment:
```
if self.item: ret[FieldName.ITEM_ID] = self.item
```